pkg/integrations/gitlab.go

```diff
@@ -38,10 +38,10 @@ func (g *GitLabIntegration) PersistentFlags(p *pflag.FlagSet) {
 
 	p.StringVar(&g.host, "host", g.host,
 		"GitLab host, defaults to 'gitlab.com'")
-	p.StringVar(&g.clientId, "app-id", g.clientId,
-		"GitLab application client id")
-	p.StringVar(&g.clientSecret, "app-secret", g.clientSecret,
-		"GitLab application client secret")
+	// p.StringVar(&g.clientId, "app-id", g.clientId,
+	// 	"GitLab application client id")
+	// p.StringVar(&g.clientSecret, "app-secret", g.clientSecret,
+	// 	"GitLab application client secret")
```

Comment:
Please remove these lines instead of just comment them out.

Reply:
Done